doc/templates/hz/cover-index.html

```diff
@@ -51,7 +51,7 @@
               </button>
               
               <a class="navbar-brand" href="../../index.html">
-                <img id="logo" src="../../images/hazelcast-white.png" alt="">
+                <img id="logo" src="https://hazelcast.com/files/brand-images/logo/hazelcast-logo.png" alt="">
```

Comment:
Is there a white logo? It doesn't look good on a dark background. Also, I'm not sure having an external link to logo. The logo can change and it may not fit like size, color etc.

Reply:
> Is there a white logo? It doesn't look good on a dark background.

https://hazelcast.com/files/brand-images/logo/hazelcast-logo-reverse.png

> Also, I'm not sure having an external link to logo. The logo can change and it may not fit like size, color etc.

I was trying to avoid hardcoded images going out of date, but to be fair the URLs don't look permanent enough to support that

Reply:
Could you change it back to hard coded than?

Reply:
> Could you change it back to hard coded than?

I can replace external links with local copies of those images if that's what you mean?

Reply:
Yes, please.

Reply:
> > Could you change it back to hard coded than?
> 
> I can replace external links with local copies of those images if that's what you mean?



> Yes, please.

7107e351ddd488db442dbed3b7dc5cf9882f59ae